src/BylawsText.js

```diff
@@ -111,8 +149,19 @@ const StyledContainer = styled.section`
 
 const StyledSmallButtonContainer = styled.div`
   display: flex;
+  gap: 20px;
```

Comment:
The gap property is not supported by all major browsers yet. You might want to look into some other way to position the items, like some margin for example.

see here https://caniuse.com/flexbox-gap

Reply:
I changed it :-)